### PR TITLE
fix #124416: Add Contributing.md

### PR DESCRIPTION
**Pull request by KrisBierma**
Resolves: *(https://musescore.org/en/node/124416)*

*(A condensed version of selected resources to help a dev new to MuseScore get started.)*

- [ x] I signed [CLA](https://musescore.org/en/cla)
- [ ] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [ ] I made sure the code compiles on my machine
- [x ] I made sure there are no unnecessary changes in the code
- [x ] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x ] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x ] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+Welcome. We're really glad you're here. [MuseScore](https://musescore.org/) is exceptional software (yes, we're biased) created, updated and used by many.
+
+It has thorough, organized documentation that contains all the resources you�ll need to help keep MuseScore great. A few of these resources are linked at the bottom. This is a shortened and condensed version to help developers new to MuseScore get started. 
+
+## Getting Started - The Basics
+
+### See MuseScore in action
+* [Install](https://musescore.org/en/handbook/3/installation) it
```
Comment (Jojo-Schmitz) [inline CONTRIBUTING.md:10]: if you drop the `3` here (plus one of the adjacent `/`) the URL will redirect to whatever version is current at the time. As this PR is for master, thus would be 4 by the time it gets released.
Same for URLs to the handbook furter down

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Make Updates
+* [Your first contribution](https://musescore.org/en/handbook/developers-handbook/getting-started/your-first-code-contribution)
+* List of all [issues](https://musescore.org/en/project/issues/musescore)
+* Use the [code guidelines](https://github.com/musescore/MuseScore/wiki/CodeGuidelines) (they're _super_ important)
+* [Compile](https://musescore.org/en/handbook/developers-handbook/compilation) the code
+* Continue following the [Suggested Workflow](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow) section to keep your branch up-to-date and create a Pull Request
+
+### Contributor Agreement
+* Sign the [CLA](https://musescore.org/en/cla)
```
Comment (Jojo-Schmitz) [inline CONTRIBUTING.md:30]: Here you might drop the `en` (and an adjacent `/`), it'll redirect properly.

**Comment by Jojo-Schmitz** [comment]
In the PR title you have a # too much, and better have `fix #124416: Add Contributing.md` as the commit title too, so the corresponding issue gets marked fixed automagically

**Comment by RomanPudashkin** [comment]
@shoogle please, check this PR

**Review by shoogle — CHANGES_REQUESTED** [review]
Thanks for the PR! However, I would prefer to see this information provided on an editable page in the GitHub Wiki rather than stored in the repository where each edit requires a submitting a new pull request.

Feel free to create a new page in the GitHub Wiki specifically for contributing, and then put a link to that page in `Contributing.md`. Putting more in Contributing.md is just unnecessary duplication and will result in it going stale.

People are most likely to be exposed to `Contributing.md` [after submitting a PR](https://github.blog/2012-09-17-contributing-guidelines/), so by this point they probably already know how to compile the program and edit the code, and they most certainly already have the release version installed. However, they might not know about specific requirements like the CLA, coding rules, or how to write a good commit message and PR title (although this is also exposed in the [PR checklist](https://github.com/musescore/MuseScore/blob/master/.github/pull_request_template.md)).

File: CONTRIBUTING.md (hunk 1 above)
Comment (KrisBierma) [reply CONTRIBUTING.md:10]: all 3s and 'en's dropped from links

File: CONTRIBUTING.md (hunk 2 above)
Comment (KrisBierma) [reply CONTRIBUTING.md:30]: all /3's and /en's dropped from links

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* Take a tour (there's an option in the program) or read a little [here](https://musescore.org/en/handbook/3/learning-musescore)
+* Try it out
+
+### Set up your environment
+* Step-by-step directions [here](https://github.com/musescore/MuseScore/wiki/Set-up-developer-environment)
+
+### Get the code
+* Check out the [Suggested Workflow](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow#Suggested_workflow)
+* You'll probably want to [fork it](https://www.toolsqa.com/git/difference-between-git-clone-and-git-fork/)
+* Here's the [GitHub repo](https://github.com/musescore/MuseScore)
```
[the remark this answers is not in the document]
Comment (KrisBierma) [reply CONTRIBUTING.md:20]: this has been dropped from wiki page

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+### Make Updates
+* [Your first contribution](https://musescore.org/en/handbook/developers-handbook/getting-started/your-first-code-contribution)
+* List of all [issues](https://musescore.org/en/project/issues/musescore)
+* Use the [code guidelines](https://github.com/musescore/MuseScore/wiki/CodeGuidelines) (they're _super_ important)
+* [Compile](https://musescore.org/en/handbook/developers-handbook/compilation) the code
+* Continue following the [Suggested Workflow](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow) section to keep your branch up-to-date and create a Pull Request
```
[the remark this answers is not in the document]
Comment (KrisBierma) [reply CONTRIBUTING.md:27]: these have been dropped

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Resource Highlights
+* [Communication](https://musescore.org/en/handbook/developers-handbook/references/developer-group-chats)
+* [Community documentation](https://musescore.org/en/community-documentation)
+    * [User Handbook](https://musescore.org/en/handbook)
+* [Developer's Handbook](https://musescore.org/en/handbook/developers-handbook)
+    * [IDEs](https://musescore.org/en/handbook/developers-handbook/ide-configuration)
+* [GPL License](https://github.com/musescore/MuseScore/blob/master/LICENSE.GPL)
+* [Wiki](https://github.com/musescore/MuseScore/wiki)
+    * [Code structure](https://github.com/musescore/MuseScore/wiki/CodeStructure )
+    * [Documentation guidelines](https://github.com/musescore/MuseScore/wiki/DocsGuidelines)
```
[the remark this answers is not in the document]
Comment (KrisBierma) [reply CONTRIBUTING.md:41]: these have been dropped from wiki page

File: CONTRIBUTING.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (KrisBierma) [reply CONTRIBUTING.md:20]: these have been dropped from wiki page

File: CONTRIBUTING.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (KrisBierma) [reply CONTRIBUTING.md:41]: these have been dropped from wiki page

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (KrisBierma) [reply CONTRIBUTING.md:27]: these have been dropped from wiki

File: CONTRIBUTING.md (hunk 2 above)
Comment (KrisBierma) [reply CONTRIBUTING.md:30]: /en and /3 dropped from links

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Contirbuting.md is available on the [wiki](https://github.com/musescore/MuseScore/wiki/Contributing.md)
```
Comment (shoogle) [inline CONTRIBUTING.md:1]: I've removed the `.md` from the end of the Wiki page so now it's just "Contributing".

Also, I think this wording is a bit clearer:

```markdown
If you would like to contribute, please see the [Contributing page](https://github.com/musescore/MuseScore/wiki/Contributing.md) on the GitHub Wiki. Thanks!
```

What do you think?

**Review by shoogle — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 6 above)
Comment (KrisBierma) [reply CONTRIBUTING.md:1]: Sounds good

Comment (Jojo-Schmitz) [inline CONTRIBUTING.md:1]: Typo, Contirbuting.md vs Contributing.md

Comment (Jojo-Schmitz) [reply CONTRIBUTING.md:1]: Also @shoogle's comment hasn't been acted upon, yet the conversation had ben maked resolved?

Comment (KrisBierma) [reply CONTRIBUTING.md]: Sorry about that. It's been changed and pushed, and commits have been squashed.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1 @@
+If you would like to contribute, please see the [Contributing page](https://github.com/musescore/MuseScore/wiki/Contributing.md) on the GitHub Wiki. Thanks!
```
Comment (Jojo-Schmitz) [inline CONTRIBUTING.md:1]: as @shoogle mentioned earlier: in the Wiki the Contribution file lost the .md extension, so it needs to here in the URL too

Comment (KrisBierma) [reply CONTRIBUTING.md]: .md removed

**Review by Jojo-Schmitz — APPROVED** [review]

File: CONTRIBUTING.md (hunk 6 above)
Comment (shoogle) [reply CONTRIBUTING.md]: Great, thanks!

**Pull request merged**